apicast/src/oauth/keycloak.lua

```diff
@@ -157,17 +163,21 @@ function _M:transform_credentials(credentials)
   local jwt_obj, err = parse_and_verify_token(self, credentials.access_token)
 
     if err then
-      return nil, err
+      return nil, nil, err
     end
 
     if jwt_obj.payload then
       local app_id = jwt_obj.payload.aud
+      local ttl
+      if jwt_obj.payload.exp then
```

Comment:
I think the condition is not necessary. All this can be handled in `timestamp_to_seconds_from_now`.